_posts/blog/2018-11-02-el-topo-blindado.md

```diff
@@ -0,0 +1,59 @@
+
+The original exploration of this contemporary medium would take the form of a blog - of the same name - that would begin the process of making these primary sources available to the general public. The blog itself functioned much like the magazine: as both a space for the continuation of the debate and as an invitation to the general public (or those with an interest) to participate. What the digital format provided that the magazine platform did not, was the ability to store and make available digital scans of the primary sources. As the archive grew, however, it became clear to the collective that the blog platform was insufficient and somewhat cumbersome, and so in 2011 the blog site transformed into the site that is being reviewed here. 
+
+The stated aims behind the creation of the new site are twofold: to deepen the study on the armed struggle in Argentina and to more effectively establish an archive of primary sources that will remain free of access to any interested persons(s). The site itself is divided into four areas: 
+
+-	Documents (the digital archive of OPM primary sources)
+-	Articles (a digital archive of secondary material)
+-	Books (a digital archive of the work produced by the collective in book format)
+-	Index (a listing of the OPM's whose documentation is presented in the site)
+
+The OPM's whose documentation is collected in this site, although far too many to name here, are organized into a classificatory system based on their political ideology/methodology. These are:
+
+-	**OPM Frentistas**: organizations whose social-construction project was based on the unity of action of the different organizations. For example, *Fuerzas Argentinas de Liberación* (FAL) and *Guerrilla of the Liberation Army* (GEL). 
+-	**OPM Marxistas**: OPMs that affirm Marxism (or one of its variants) as the point of departure for their tactics, strategies and political thinking. For example, *Baluarte, Comando Santiago Pampillón* (CSP), *Ejército Guerillero del Pueblo* (EGP), *Ejército Revolucionario del Pueblo* (ERP), *Partido Revolucionario de los Trabajadores* (PRT), among many others.
+-	**OPM Peronistas**: various organizations of Peronist origin. For example, *Acción Revolucionaria Peronista, Resistencia Peronista, Movimiento Revolucionario Peronista*, various student led organizaciones (juventudes), etc. 
```

Comment:
Rather than listing each of these, summarize the content in a single paragraph: The documentation collected in this site is organized by the political ideology of each OPM, broadly defined as Frentista (definition), Marxista (definition), or Peronista (definition). 

You could also include a sentence of your thoughts here: does this seem comprehensive? Is this method of categorizing useful?